mw_url_rewrite.go

```diff
@@ -17,17 +17,14 @@ import (
 )
 
 func urlRewrite(meta *apidef.URLRewriteMeta, r *http.Request) (string, error) {
-	// Find all the matching groups:
-	mp, err := regexp.Compile(meta.MatchPattern)
-	if err != nil {
-		log.Debug("Compilation error: ", err)
-		return "", err
+	if meta.MatchRegexp == nil {
+		return "", fmt.Errorf("URLRewrite regexp error", meta.MatchPattern)
```

Comment:
we need to add placeholder into format string - this one was caught by my IDE :)